main.tf

```diff
@@ -207,12 +207,25 @@ resource aws_vpn_gateway_route_propagation "vpn_routes_private" {
 # Flow Logs
 ###########
 
+resource aws_flow_log "vpc_log" {
```

Comment:
I would prefer to have only a single `aws_flow_log` resource unless there is a reason to have them separated, such as incompatible options.  It looks like the same parameters are being used in both resources.  We are not explicitly decaring the `log_destination_type` for the CWLogs resource, but could easily have all parameters values present and set based on a choice.

If we do need the two resources separated, then we should add in a few comments as to why and then explicitly name this one as the S3 flow log resource, as `vpc_log` is ambiguous.

Reply:
I would prefer to have them separated , the ability is present to both active flow logs to cloudwatch and bucket.
This also mean upgrading to later module version does not require to restructure the arguments. 
If we change to only having one aws_flow_log , we can only ship to one location at a time.